scripts/allowedTokens

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-NETWORK_ID=$2
+NETWORK_ID=$1
```

Comment:
I would rename this variable to `NETWORK_NAME` since that it's not anymore a network id

Reply:
agreed

Reply:
Fixed.